app/main.py

```diff
@@ -4,4 +1,26 @@
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+API_KEY = os.getenv("API_KEY")
+URL = "http://api.weatherapi.com/v1/current.json?"
+CITY = "Paris"
+
+
+def get_weather(city: str, key: str) -> None:
+    try:
+        response = requests.get(f"{URL}key={API_KEY}&q={CITY}&aqi=no")
+        response.raise_for_status()
+        data = response.json()
+        print(f"{data["location"]["name"]}/{data["location"]["country"]} " # noqa
+              f"{data["current"]["last_updated"]} "
+              f"Weather: {data["current"]["temp_c"]} Celsius, "
+              f"{data["current"]["condition"]["text"]}")
```

Comment:
The use of double quotes inside the f-string is causing a syntax error. Consider using single quotes for the string literals inside the f-string to avoid this issue.